### PR TITLE
update

### PR DESCRIPTION
**Pull request by cike-567**
<merge>(<update>): <from {base branch name} into {target branch name}>
<BLANK LINE(Not necessary)>
<body>

### CODE REVIEW

File: source/4-acpi.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 title: 4. ACPI
 description: 加载、屏蔽、修补 ACPI（DSDT/SSDT）表
 type: docs
-author_info: 由 Sukka 整理、由 Sukka、derbalkon、EricKwok 翻译。感谢黑果小兵提供的参考资料
+author_info: 由 Sukka cike-567整理、由 Sukka、derbalkon、EricKwok cike-567翻译。感谢黑果小兵提供的参考资料
```
Comment (SukkaW) [inline source/4-acpi.md]:
```suggestion
author_info: 由 Sukka、cike-567 整理、由 Sukka、derbalkon、EricKwok、cike-567 翻译。感谢黑果小兵提供的参考资料
```

**Review by SukkaW — CHANGES_REQUESTED** [review]

File: source/10-platform-info.md [hunk 2]
```diff
@@ -676,51 +699,51 @@ OpenCore 在生成修改过的 DMI 表时，总是设置最新的 SMBIOS 版本
...
 **Description**: 机箱类型名称。有各种各样，可以留空或填 `MacBook-Aluminum`。
 
 ### 23. `PlatformFeature`
 
 **Type**: `plist integer`, 32-bit
-**Failsafe**: `0xFFFFFFFF`
+**Failsafe**: `0xFFFFFFFF`（在苹果硬件上指定的 OEM，否则不提供表）
 **SMBIOS**: `APPLE_SMBIOS_TABLE_TYPE133` - `PlatformFeature`
-**Description**: 平台功能位掩码，详见 [AppleFeatures.h](https://github.com/acidanthera/OpenCorePkg/blob/master/Include/Apple/IndustryStandard/AppleFeatures.h)。填写 `0xFFFFFFFF` 值时不提供此表。
+**Description**: 平台功能位掩码（在较旧的Mac上缺失），详情请参考 [AppleFeatures.h](https://github.com/acidanthera/OpenCorePkg/blob/master/Include/Apple/IndustryStandard/AppleFeatures.h)。
```
Comment (SukkaW) [inline source/10-platform-info.md:725]:
```suggestion
**Description**: 平台功能位掩码（较旧的 Mac 上缺失该位掩码），详情请参考 [AppleFeatures.h](https://github.com/acidanthera/OpenCorePkg/blob/master/Include/Apple/IndustryStandard/AppleFeatures.h)。
```

File: source/4-acpi.md [hunk 3]
```diff
@@ -2,14 +2,14 @@
 title: 4. ACPI
 description: 加载、屏蔽、修补 ACPI（DSDT/SSDT）表
 type: docs
-author_info: 由 Sukka 整理、由 Sukka、derbalkon、EricKwok 翻译。感谢黑果小兵提供的参考资料
-last_updated: 2021-04-17
+author_info: 由 Sukka、cike-567整理、由 Sukka、derbalkon、EricKwok、cike-567翻译。感谢黑果小兵提供的参考资料
```
Comment (SukkaW) [inline source/4-acpi.md:5]:
```suggestion
author_info: 由 Sukka、cike-567 整理、由 Sukka、derbalkon、EricKwok、cike-567 翻译。感谢黑果小兵提供的参考资料
```

**Review by SukkaW — CHANGES_REQUESTED** [review]

**Comment by cike-567** [comment]
你可以说中文的

**Pull request closed without merging**